statsd/datadog_checks/statsd/statsd.py

```diff
@@ -21,11 +21,12 @@ class StatsCheck(AgentCheck):
     def check(self, instance):
         host = instance.get("host", "localhost")
         port = instance.get("port", 8126)
+        timeout = int(instance.get('timeout', 10))
```

Comment:
Why not casting to `float` here and avoid another cast at line 67?

Reply:
Good point! Done, and fixup on the same commit.